### ISSUE TITLE
Default swatches won't re-appear

### ISSUE THREAD

**Issue by smsunday86** [Loupimo/UniversalSwatchSlots#6]
So in experementing with this mod, I accidently deleted all of the Swatches that came with it by default. I followed you instructions about deleting the JSON file. but it never seems to put pack the default swatches. I've even attempted to uninstall the mod, delete the JSON file, start/stop satisfactory, then re-install the mod. That did not bring back the defaults.

**Comment by Loupimo** [comment]
I'm going to check that, thanks for the report.

Meanwhile here is the default config : 
[UniversalSwatchSlots.zip](https://github.com/user-attachments/files/17836557/UniversalSwatchSlots.zip)

**Comment by smsunday86** [comment]
Okay, so I found my issue with the default config, it was user error on my side. I was looking in the wrong directory. However I am having a new issue that is confusing me. 
I add my session name to the USS Default - English Pallet as seen below:
![Screenshot 2024-11-20 161834](https://github.com/user-attachments/assets/c5631e42-aede-443f-ae54-007e57ef2e28)
but when I get into the game, I have the french pallet, and it only give me 11 of the 18 pallets.
![Screenshot 2024-11-20 162041](https://github.com/user-attachments/assets/ad169fcd-2f8e-48dd-a6d3-c3a4a29b8308)
Let me know if you need any futher information.

**Comment by Loupimo** [comment]
Yeah that's a problem that has been already reported and I'm working on it.
As a workaround just leave a blank in the session name field (however note that the palette will be applied for every session you load)

**Comment by Loupimo** [comment]
Hi there,

New version available ! I hope it will solved most of your issue.

Please note that you have already applied will break (like 99% sure) but it should be the last time as I'm saving info about previously loaded swatches now :)

Group and swatches should show up normally now 🤞
